Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1325 @@
+                              device_name: str, process_name: str, sort_field: str, sort_order: str,
+                              limit: int) -> dict:
+
+        suffix_url = f'/appservices/v6/orgs/{CB_ORG_KEY}/alerts/_search'
+        body = {
+            'criteria': assign_params(
+                group_results=group_results,
+                minimum_severity=minimum_severity,
+                create_time=create_time,
+                device_os_version=device_os_version,
+                policy_id=policy_id,
+                tag=alert_tag,
+                id=alert_id,
+                device_username=device_username,
+                device_id=device_id,
+                device_os=device_os,
+                process_sha256=process_sha256,
+                policy_name=policy_name,
+                reputation=reputation,
+                type=alert_type,
+                category=alert_category,
+                workflow=workflow,
+                device_name=device_name,
+                process_name=process_name
+            ),
+            'sort': [
+                {
+                    'field': sort_field,
+                    'order': sort_order
+                }
+            ],
+            'rows': limit,
+            'start': 0
+        }
+        return self._http_request('POST', suffix_url, json_data=body)
+
+    def alert_workflow_update_request(self, alert_id: str, state: str, comment: str,
+                                      remediation_state: str) -> Dict[str, Any]:
+
+        suffix_url = f'/appservices/v6/orgs/{CB_ORG_KEY}/alerts/{alert_id}/workflow'
+        body = assign_params(
+            state=state,
+            comment=comment,
+            remediation_state=remediation_state
+        )
+
+        return self._http_request('POST', suffix_url, json_data=body)
+
+    def devices_list_request(self, device_id: Union[list, str], status: str, device_os: str,
+                             last_contact_time: Dict[str, Optional[Any]], ad_group_id: int, policy_id: int,
+                             target_priority: str, limit: int, sort_field: str, sort_order: str) -> dict:
+
+        suffix_url = f'/appservices/v6/orgs/{CB_ORG_KEY}/devices/_search'
+
+        body = {
+            'criteria': {
+                'id': device_id,
+                'status': status,
+                'os': device_os,
+                'last_contact_time': last_contact_time,
+                'ad_group_id': ad_group_id,
+                'policy_id': policy_id,
+                'target_priority': target_priority
+            },
+            'rows': limit,
+            'start': 0,
+            'sort': [
+                {
+                    'field': sort_field,
+                    'order': sort_order
+                }
+            ]
+        }
+
+        return self._http_request('POST', suffix_url, json_data=body)
+
+    def device_quarantine_request(self, device_id: Union[list, str]) -> None:
```

Comment:
you pass to `device_id` the ouput for `argToList`, so this will never be a `str`, but only `List`